### PR TITLE
Ubuntu installation instructions

### PR DESCRIPTION
**Pull request by kvmanohar22**
Adding installation instructions for ubuntu.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,8 +76,13 @@ $ ls opencv2
 ```
 
 
+#### Linux (Ubuntu)
+See links [1](http://docs.opencv.org/trunk/doc/tutorials/introduction/linux_install/linux_install.html) and [2](https://github.com/jayrambhia/Install-OpenCV) on how to install OpenCV `3.2.0`.
+Comment out the #include statements for `flann` in `opencv2/opencv.hpp`.
+
+
```
Comment (maxruby) [inline README.md]: Please remove the entire section below `Windows and Linux`.  We will add notes on support for Windows when it becomes available.  Thanks.

**Comment by vernwalrahul** [comment]
@kvmanohar22  Can you add the installation instruction or link for julia 0.6.0?

**Comment by kvmanohar22** [comment]
@vernwalrahul  you need to check out `v0.6.0-pre.alpha` branch before you build julia.

**Comment by vernwalrahul** [comment]
@kvmanohar22 thanks, but that branch seems to be inactive. I am getting the following error:
HEAD is now at 29bc2ac... feature freeze master for 0.6-pre (#20762)

and its jumping back to release 0.5.

**Pull request merged**